### PR TITLE
Changed saving throw bonuses to use bonus table not melee table

### PR DESCRIPTION
**Pull request by vandervecken**
https://github.com/kevL/TccScripts/issues/45

### CODE REVIEW

**Comment by kevL** [comment]
i'm skeptical, does this fix or even change any game mechanic? Or is it simply something that should have been laid out this way, by Obsidian, from the git-go?

Am not against it, strictly speaking, because it makes sense at least superficially. But i'm concerned about introducing changes to game-files that result in no practical difference for the Reboot itself ...

I mean, if it causes crafting to throw a "invalid itemproperty" for recipes that try to apply a save-bonus of >12 then i'm for it ... i guess ( since save-bonus >12 doesn't appear to work IG )

in a nutshell:
Can someone tell me what the practical purpose of this change is? Preferably exactly or with an example,

**Comment by vandervecken** [comment]
@zzcmeiou do you want to try to test this branch with your recipe for >+12 bonus? I expect it will throw an error now.
